server/scripts/seed_sample_data.js

```diff
@@ -29,25 +39,37 @@ if (DATA.indexOf('.json') === -1) {
 }
 
 const { records, agents } = require(`./${DATA}`)
-let batcherPubkey = null
+const context = new secp256k1.Secp256k1Context()
+let batcherPublicKey = null
+
+const encodeHeader = (signerPublicKey, payload) => {
```

Comment:
Seems like this could be pulled out into a library file, for all the of the server code.